doc/src/manual/methods.md

```diff
@@ -335,7 +336,20 @@ Here's an example where the method type parameter `T` is used as the type parame
 type `Vector{T}` in the method signature:
 
 ```jldoctest
-julia> myappend(v::Vector{T}, x::T) where {T} = [v..., x]
+julia> function myappend(v::Vector{T}, x::T) where {T}
+           return [v..., x]
+       end
+myappend (generic function with 1 method)
+```
+
+The type parameter `T` in the function's signature ensures that the type of the new `v` is preserved after element `x` is appended to it. 
+The function's signature is preceded by the keyword `where` and is placed immediately after the method arguments. 
```

Comment:
I think it is weird to explain this here, since the `where` syntax has been used since the beginning of the section.